app/Services/ReportbackService.php

```diff
@@ -28,4 +36,37 @@ public function create($data)
 
         return $reportback;
     }
+
+    /*
+     * Handles all the business logic around updating a reportback.
+     *
+     * @param \Rogue\Models\Reportback $reportback
+     * @param array $data
+     *
+     * @return \Rogue\Models\Reportback $reportback.
+     */
+    public function update($reportback, $data)
+    {
+        $reportback = $this->reportbackRepository->update($reportback, $data);
+
+        return $reportback;
+    }
```

Comment:
yeah, the Service here isn't really doing much to help out the Repository... just acting as a middleman. You can probs inject the Repository in the `ReportbackController` along w/ the Service for where that's needed instead only passing the Service and needing duplicate methods like this one to reach the Repository... 

I might be missing something though. Thoughts?